inferlo/pairwise/optimization/map_lp.py

```diff
@@ -0,0 +1,78 @@
+from collections import namedtuple
+import cvxpy as cp
+
+from inferlo import PairWiseFiniteModel
+
+map_lp_result = namedtuple('map_lp_result', ['upper_bound',
+                                             'lower_bound',
+                                             'node_beliefs',
+                                             'edge_beliefs',
+                                             'normalization_duals',
+                                             'marginalization_duals'
+                                             ])
+
+
+def map_lp(model: PairWiseFiniteModel) -> map_lp_result:
+    """MAP linear programming relaxation for
+            normal factor graph with finite alphabet
```

Comment:
This identation looks weird.

Please follow these guidelines for function docstrings:
https://google.github.io/styleguide/pyguide.html#383-functions-and-methods